### PR TITLE
:sparkles: Real time censoring on macOS

### PR DESCRIPTION
**Pull request by levin-noro**
Got rid of a lot of repetitive functions from earlier versions.
Not sure what to do about the duration of recording right now. It currently set to a default of 60 second. Will the real-time pausing work item take care of this? or should we allow users to specify durations as an argument for now?

### CODE REVIEW

File: cleansio/utils/cleanup.py [hunk 1]
```diff
@@ -7,13 +7,17 @@
...
 from .files import append_before_ext
+import subprocess
 
 # Cleans up files on normal or abnormal exit
 # The arguments are unused - they are only here to satisfy atexit.
 def cleanup(_sig_num=None, _cur_stack_frame=None):
     """ Removes temporary files """
     remove_conversions()
     remove_chunks()
+    if 'CLEANSIO_REALTIME' in environ and 'CLEANSIO_OLD_SOUND_OUT' in environ and 'CLEANSIO_OLD_SOUND_IN' in environ:
```
Comment (PatrickDuncan) [inline cleansio/utils/cleanup.py]: Please split if into multiple lines

File: cleansio/utils/cleanup.py [hunk 2]
```diff
@@ -7,13 +7,17 @@
...
+        subprocess.run(['SwitchAudioSource', '-t', 'output', '-s', environ['CLEANSIO_OLD_SOUND_OUT']])
```
Comment (PatrickDuncan) [inline cleansio/utils/cleanup.py]: We need a mac cleanup class

File: cleansio/censor/censor_realtime.py [hunk 3]
```diff
@@ -1,14 +1,104 @@
 """ Censors audio chunks in a continuous stream """
```
Comment (PatrickDuncan) [inline cleansio/censor/censor_realtime.py]: There needs to be a mac class for realtime censoring. I have most of it done, I'll add it with you in person

File: cleansio/audio/convert.py [hunk 4]
```diff
@@ -27,3 +27,25 @@ def convert(file_path, encoding='wav'):
         str(milliseconds) + '.' + encoding
     __create_converted_file(file_path, encoding)
     return os.environ['CLEANSIO_TEMP_FILE']
+
+def read_and_convert_audio(file_path, encoding):
+    audio_segment = AudioSegment.from_file(file_path)
+    audio_segment                                     \
+        .set_channels(1)                              \
+        .set_sample_width(2)                          \
+        .set_frame_rate(__sample_rate(audio_segment))
```
Comment (PatrickDuncan) [inline cleansio/audio/convert.py]: read_and_convert_audio and convert_audio_segment and use a shared private function

**Comment by levin-noro** [comment]
Superseded by pull request #146

**Pull request closed without merging**